stories/Button/Guideline.mdx

```diff
@@ -0,0 +1,169 @@
+import {
+  Meta,
+  Story,
+  Canvas,
+  Primary,
+  Controls,
+  Unstyled,
+  Source,
+} from '@storybook/blocks';
+import { Button } from '../../src/lib/components/buttonv2/Buttonv2.component';
+
+import * as ButtonStories from './button.stories';
+
+<Meta of={ButtonStories} />
+
+# Button
+
+Buttons are used to trigger an action or event when clicked.
+
+## Size & style
+
+### Style
+
+- Height: 2rem
+- Border-radius: 3px
+- Minimum width: 5rem
```

Comment:
nit: Maybe we want to unify the way we prensent those style (Border radius without the hyphens or use the hyphen notation for the width), probably we should pick the without hyphens notation to be aligned with the padding

Reply:
Technical question: the height and the border-radius are part of the component, but what about the minimum width? Is it enforced somewhere in the code, or it's more guideline relative?
Moreover, I would say that the minimum-width is valid for standard-size buttons, not for "inline" buttons.

Reply:
The min width can be and should be enforced in the code IMO.